CONTRIBUTING.MD

```diff
@@ -62,22 +69,18 @@ Code Checking with [Flake8](https://github.com/PyCQA/flake8)
 - Combines PyFlakes, pycodestyle, and others for linting and style checking.
```

Comment:
Are these standards or Developer tools? How necessary is this if we have them previously mentioned in the contributing steps above?

Reply:
Removed this part of contributing.